### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by simonmerckx** [DavidLee-4032/Graph_Opt#4]
Which hyperparameters did you use to obtain your final approximation ratio? I can't seem to get the same results on MVC problem and the approximation ratio even seems to be increasing with minibatch: 128 and n-step: 5 on a validation set of 100 examples.
